File: CommandLine.c
#include<stdio.h>
#include<stdlib.h>

 // int main(int argc, char *argv[], int *envp)     envp environment pointer

int main(int argc, char *argv[])
{
    printf("Demonstration of command line\n");
    
    int No1 = atoi(argv[1]);
    int No2 = atoi(argv[2]);

    int Ans = No1 + No2;

    printf("Number of command line arguments: %d\n",argc);
    printf("Addition is : %d\n",Ans);

    return 0;
}

// gcc CommandLine.c -o Myexe

File: CommandLine.java
class CommandLine
{
    public static void main(String arg[])
    {
        System.out.println(" Demonstration of command line");

        int No1 = Integer.parseInt(arg[0]);
        int No2 = Integer.parseInt(arg[1]);

        int Ans = No1 + No2;
        System.out.println("Number of command line arguments"+arg.length);

        System.out.println("Addition is :"+Ans);
    }
}

// javac CommandLine.java
// java CommandLine 11 21